src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/Validators/WageTypeValidator.cs

```diff
@@ -102,13 +138,33 @@ private void ConfigureWageTypeValidator()
                 RuleFor(request => request.WageTypeReason)
                     .NotEmpty()
                     .WithErrorCode(ErrorCodes.CreateApprenticeship.WageTypeReason);
+
+                RuleFor(request => request.WageUnit)
+                    .Equal(WageUnit.NotApplicable)
+                    .WithErrorCode(ErrorCodes.CreateApprenticeship.WageUnit);
             });
 
             RuleFor(request => request.WageTypeReason)
-                .MaximumLength(240)
+                .MaximumLength(wageTypeReasonMaxLength)
                 .WithErrorCode(ErrorCodes.CreateApprenticeship.WageTypeReason)
                 .MatchesAllowedFreeTextCharacters()
                 .WithErrorCode(ErrorCodes.CreateApprenticeship.WageTypeReason);
         }
+
+        private async Task<bool> BeGreaterThanOrEqualToApprenticeshipMinimumWage(CreateApprenticeshipRequest request, WageUnit wageUnit, CancellationToken cancellationToken)
```

Comment:
I would have preferred this validation taking place in the handler separately like we are validating the Vacancy Owner link. This could have saved refactoring all the test classes as well reducing impact heavily.